pkgs/applications/misc/easystroke/default.nix

```diff
@@ -0,0 +1,31 @@
+# based on easystroke-git from Arch AUR
+# as found on https://github.com/teu5us/easystroke-nix
+
+{ pkgs, ... }:
+
+with pkgs;
```

Comment:
using `{ stdenv, gnutar, <other dependencies> }:` is preferred since it allows overriding.

Reply:
actually using with pkgs and pkgs as inputs is in packages not allowed.